src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -27,6 +27,8 @@ public class MicrobeEditor : Node, ILoadableGameState, IGodotEarlyNodeResolve
     [JsonProperty]
     public float CurrentOrganelleCost;
 
+    private Vector3 arrowPosition = Vector3.Zero;
```

Comment:
Should this be saved as well or not needed?

Reply:
ArrowPosition does not need to get saved because it gets calculated in InitEditor